tests/mysql/test-mysql.cpp

```diff
@@ -297,7 +297,7 @@ void test_num(const char* s, bool valid, T value)
         {
             char const * expectedPrefix = "Cannot convert data";
             CAPTURE(e.what());
-            CHECK(strncmp(e.what(), expectedPrefix, strlen(expectedPrefix)) == 0);
+            CHECK( (strncmp(e.what(), expectedPrefix, strlen(expectedPrefix)) == 0) );
```

Comment:
`test_eq()` would break CATCH expression decomposition. In the particular case of `strncmp() == 0` it doesn't matter anyhow because decomposition doesn't work with it anyhow, but for anything else it would be a regression.

From the point of view of decomposition, the best solution is actually
```
CHECK( s.substr(0, strlen(expected)) == expected );
```
And if we do this, we could get rid of `CAPTURE(s)` before it.